CHANGELOG.rst

```diff
@@ -1,6 +1,60 @@
 Changelog
 =========
 
+0.6.0 (2018-09-10)
+------------------
+
+MLflow 0.6.0 introduces two major features:
+
+- A Java client API, published to Maven
+- Inherent support for saving and serving SparkML models as MLeap for low-latency serving
+
+In addition to these features, there are a host of improvements and bugfixes to the REST API, Python API, tracking UI, and documentation.
+
+Breaking changes:
+
+We fixed a few inconsistencies in the the ``mlflow.tracking.MlflowService`` introduced in 0.5.0:
+
+- ``MlflowService.list_runs`` was changed to ``MlflowService.list_run_infos`` to reflect the information actually returned by the call. It now returns a ``RunInfo`` instead of a ``Run`` (#334, @aarondav)
+- ``MlflowService.log_artifact`` and ``MlflowService.log_artifacts`` now take a ``run_id`` instead of ``artifact_uri``. This now matches ``list_artifacts`` and ``download_artifacts``  (#444, @aarondav)
+
+Features:
+
+- Java client API added with support for the MLflow Tracking API (analogous to ``mlflow.tracking``), allowing users to create and manage experiments, runs, and artifacts. The release includes a `usage example <https://github.com/mlflow/mlflow/blob/master/mlflow/java/client/src/main/java/org/mlflow/tracking/samples/QuickStartDriver.java>`_ and `Javadocs <https://mlflow.org/docs/latest/java_api/index.html>`_. The client is published to Maven under `mlflow:mlflow <https://mvnrepository.com/artifact/mlflow/mlflow>`_ (#380, #394, #398, #409, #410, #430, #452, @aarondav)
+- SparkML models are now saved in MLeap format (https://github.com/combust/mleap), when applicable. Model serving platforms can choose to serve using this format instead to dramatically decrease prediction latency. SageMaker now does this by default (#324, #327, #331, #395, #428, #435, #438, @dbczumar)
```

Comment:
Are they always saved in this format or are they *also* saved in this format in addition to what we had before? It's not clear when this happens or how it would affect previous users of Spark ML (e.g. what if I want to load the model back into Spark). Clarify the note a bit and maybe link to docs for it.

Reply:
Added "also". @dbczumar do we have docs for the MLeap support?